docs/man/git-lfs-config.5.ronn

```diff
@@ -190,6 +184,22 @@ lfs option can be scoped inside the configuration for a remote.
   If set to "basic" then credentials will be requested before making batch
   requests to this url, otherwise a public request will initially be attempted.
 
+* `lfs.<url>.skiplocksverify`
```

Comment:
Per our discussion in 🍖 -space, I think calling this `lfs.<url>.locks_verify` would be more appropriate, since it's one less cycle to figure out whether or not this performs the lock verification step.